app/javascript/components/public-dashboard/profile.vue

```diff
@@ -30,35 +30,25 @@
 </template>
 
 <script>
-import { mapState } from 'vuex';
+import { mapGetters } from 'vuex';
 
-import { CREATE_PUBLIC_USER_ENTRY } from '../store/mutation-types';
-import { COMPUTE_USERS_ORGANIZATION_WIDE_SCORE } from '../store/action-types';
+import { COMPUTE_USERS_ORGANIZATION_WIDE_SCORE } from '../../store/action-types';
```

Comment:
'COMPUTE_USERS_ORGANIZATION_WIDE_SCORE' is defined but never used  no-unused-vars